docs/content/get-started/aperture-cloud/sign-up.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Sign Up
+keywords:
+  - sign-up
+  - aperture-cloud
+sidebar_position: 1
+---
+
+```mdx-code-block
+import Zoom from 'react-medium-image-zoom';
+```
+
+:::info
+
+Aperture Cloud provides a 30-day free trial period. You can use all the features
+of Aperture Cloud during the trial period. After the trial period, you can
+choose to continue using Aperture Cloud by subscribing to one of the [plans][].
+
+:::
+
+To begin using Aperture Cloud, you need to sign up and create an organization &
```

Comment:
I think we don't have to mention the project explicitely here as it gets created as part of the organization creation process?